Sources/HTTP/Serializer/ResponseSerializer.swift

```diff
@@ -0,0 +1,42 @@
+import Transport
+
+public final class ResponseSerializer<Stream: WriteableStream>: Serializer {
+    typealias StreamType = Stream
+    let stream: Stream
+    public init(_ stream: Stream) {
+        self.stream = stream
+    }
+    
+    public func serialize(_ response: Response) throws {
+        let message = try serialize(response as Message)
+        
+        let reasonPhrase = response.status.reasonPhrase.makeBytes()
+        let length = 5 // HTTP/
```

Comment:
let's use some named constants for this whole row, I think the comments are good, but having something static would be better.